buildSrc/src/main/kotlin/io/spine/internal/Repositories.kt

```diff
@@ -0,0 +1,93 @@
+package io.spine.internal
+
+import org.gradle.api.artifacts.dsl.RepositoryHandler
+import org.gradle.api.artifacts.repositories.MavenArtifactRepository
+import java.net.URI
+
+
+/**
+ * Password credentials for a Maven repository.
+ */
+data class Credentials(
```

Comment:
The standard thing for our projects is to turn on Kotlin's [explicit API mode](https://kotlinlang.org/docs/whatsnew14.html#explicit-api-mode-for-library-authors).

Please have a read, and make sure it is turned on, wherever possible.